qiskit_ibm_runtime/options/options.py

```diff
@@ -81,7 +81,11 @@ class Options:
             :class:`SimulatorOptions` for all available options.
     """
 
-    optimization_level: int = 3
+    # Defaults for optimization_level and for resilience_level will be assigned
+    # in Sampler/Estimator
+    _DEFAULT_OPTIMIZATION_LEVEL = 3
+    _DEFAULT_RESILIENCE_LEVEL = 1
+    optimization_level: Optional[int] = None
     resilience_level: int = 1
```

Comment:
`resilience_level` should also be optional right?

Reply:
Certainly. It was actually this way in my clone, but I guess I forgot to commit.